scripts.js

```diff
@@ -0,0 +1,163 @@
+function validateLoginForm() {
```

Comment:
Skoro używamy bootstrapa to moglibyśmy używać wbudowanym w nim opcji walidacji formularzy zamiast pisac własne.
Na stronie to tylko tyle czy jest pole uzupełnione, a dopiero w kontrolerze czy dane są zgodne z założeniami (regexy itp)